src/main.rs

```diff
@@ -15,6 +17,32 @@ use termion::screen::IntoAlternateScreen;
  * @return Result
  */
 fn main() -> Result<(), Box<dyn Error>> {
+
+	let raw_output = io::stdout().into_raw_mode()?;
+	raw_output.suspend_raw_mode()?;
```

Comment:
Why exactly is this needed?

Reply:
As it's mentioned in the issue in ratatui, it creates a raw terminal, and immediately exit raw mode, so that the raw terminal has the correct `prev_ios` field set.

Reply:
Ah okay, I missed that :)